Web/src/components/create-room/CreateRoomPage.tsx

```diff
@@ -95,6 +104,20 @@ const CreateRoomPage: React.FC<{ commHooks: CommAttributes }> = ({ commHooks })
             Unable to get permissions to play Pinda.
           </ErrorText>
         )}
+      {waitingForResponse
+        && (
+          <ReactLoading
```

Comment:
Not sure why not use our common `Loading` component?

Reply:
Hmm, I just wanted to show like progress on the same page to indicate that there's some operations running currently. Should we change it to the loading component, or remove completely?

ping @hzxcaryn

Reply:
I think it's fine to do this then, since I assume it's not really the full screen?